src/databricks/labs/ucx/assessment/clusters.py

```diff
@@ -192,6 +192,16 @@ def _maybe_direct_owner(self, record: ClusterInfo) -> str | None:
         return record.creator
 
 
+class ClusterDetailsOwnership(Ownership[ClusterDetails]):
```

Comment:
where is it coming from? we don't have a concept of `cluster details` in our object model - we just have a cluster.